src/Avalonia.Base/Threading/DispatcherOperation.cs

```diff
@@ -103,7 +105,7 @@ public event EventHandler Completed
                 _completed += value;
             }
         }
-        
```

Comment:
Whitespace changes

Reply:
Unfortunately whitespace changes pollute `git blame` output.

We are planning to somewhat solve this in one go later, but for now please remove whitespace changes.